resty-redis-rate-1.0.0-0.rockspec

```diff
@@ -0,0 +1,23 @@
+package = "resty-redis-rate"
+version = "1.0.0-0"
+source = {
+  url = "github.com/leandromoreira/nginx-lua-redis-rate-measuring",
+  tag = "1.0.0"
+}
+description = {
+  summary = "A resty Lua library to provide distributed rate measurement using nginx + redis",
```

Comment:
I suggest change the summary to _"A resty Lua library to provide distributed rate measurement using Redis"_ since the name "resty" already suggests that the library is related to Nginx.